src/components/common/Actions.vue

```diff
@@ -189,8 +189,7 @@ export default class Actions extends Mixins(DateMixin, FilingTemplateMixin, Lega
       // redirect to pay and return to the dashboard
       const authUrl = sessionStorage.getItem('AUTH_URL')
       const dashboardUrl = sessionStorage.getItem('DASHBOARD_URL')
-      const nrNumber = this.$route.query.nrNumber as string
-      const returnUrl = encodeURIComponent(dashboardUrl + nrNumber)
+      const returnUrl = encodeURIComponent(dashboardUrl + this.getBusinessIdentifier)
```

Comment:
So are we calling the temp reg number "tempId", "temporarId" or "BusinessIdentifier"?

Reply:
I'll go with TempId :)